### PR TITLE
Change: Add reactive lots list [LEGACY]

### PR DESCRIPTION
**Pull request by RubenPX**
>> IMPORTANT: https://github.com/eReuse/devicehub-teal/pull/218#discussion_r847244127

TODOS:
- [X] Implement reactive lots list
- [x] [ISSUE] User token not working `{{ current_user.token }}`, Search alternative
- [x] ~Remove unnecesary views~
- [x] Add select all button
- [X] Re-render table when lots are changed

https://github.com/eReuse/devicehub-teal/pull/218#pullrequestreview-938087970
- [X] UX Make lot name clickable (instead of only related checkbox) https://github.com/eReuse/devicehub-teal/pull/218/commits/f934aa8f1fd1467388216e902b46eabda7291cab
- [X] Update number of existing entries on table footer https://github.com/eReuse/devicehub-teal/pull/218/commits/3ab7e8796f92ec562797c50a98ea0162d17dfbdb

### CODE REVIEW

File: ereuse_devicehub/templates/inventory/device_list.html [hunk 1]
```diff
@@ -400,6 +397,13 @@ <h5 class="card-title">Documents</h5>
 <!-- Custom Code -->
 <script>
   const table = new simpleDatatables.DataTable("table")
+  const API_URLS = {
+    Auth_Token: "Basic ZDc5YjQyOGItZTJjMy00NDA0LThhODItNzJjMTNmOWNkYmNjOg==", // {{ current_user.token }}
```
Comment (RubenPX) [inline ereuse_devicehub/templates/inventory/device_list.html:401]: `{{ current_user.token }}`

This variable returns user id instead user token

**Comment by RubenPX** [comment]
https://github.com/eReuse/devicehub-teal/pull/218/commits/184d22ac816dd9be908b5c46294bb72d52670610

Token encoded to base64 but not working

File: ereuse_devicehub/templates/ereuse_devicehub/base_site.html [hunk 2]
```diff
@@ -220,4 +220,14 @@ <h6>{{ current_user.get_full_name }}</h6>
   </div>
 </footer><!-- End Footer -->
 
+<script>
+  const API_URLS = {
```
Comment (RubenPX) [inline ereuse_devicehub/templates/ereuse_devicehub/base_site.html:224]: WARNING: This pull request maybe has a conflict with https://github.com/eReuse/devicehub-teal/pull/219

File: ereuse_devicehub/static/js/main_inventory.js [hunk 3]
```diff
@@ -180,3 +180,264 @@ function export_file(type_file) {
...
+
+    class Actions {
+
+        constructor() {
+            this.list = []; // list of petitions of requests @item --> {type: ["Remove" | "Add"], "LotID": string, "devices": number[]}
+        }
+
+        /**
+         * Manage the actions that will be performed when applying the changes
+         * @param {*} ev event (Should be a checkbos type)
```
Comment (slamora) [inline ereuse_devicehub/static/js/main_inventory.js:264]:
```suggestion
         * @param {*} ev event (Should be a checkbox type)
```

File: ereuse_devicehub/static/js/main_inventory.js [hunk 4]
```diff
@@ -180,3 +180,264 @@ function export_file(type_file) {
         $("#exportAlertModal").click();
     }
 }
+
+
+
+/**
+ * Añade reactividad al botón de lotes
```
Comment (slamora) [inline ereuse_devicehub/static/js/main_inventory.js:187]:
```suggestion
 * Reactive lots button
```

**Review by slamora — CHANGES_REQUESTED** [review]
Thanks @RubenPX almost it's working properly but after adding or removing devices of the current lot number of entries should be updated (see attached screen):

Only 1 entry exists but still remains old value "1 of 2".

- [ ] Update number of existing entries on table footer
image

- [ ] UX Make lot name clickable (instead of only related checkbox)
image

**Comment by slamora** [comment]
Superseeded by #220

**Pull request closed without merging**